scripts/release-tasks.ts

```diff
@@ -151,14 +151,9 @@ export function runReleaseTasks(opts: BuildOptions, args: ReadonlyArray<string>)
       },
       {
         title: `Set package.json version to ${color.bold.yellow(opts.version)}`,
-        task: () => {
```

Comment:
I ran the pre-release script for this, then verified the version was correct in `package.json` and `package-lock.json`

`package.json`:
```diff
 {
   "name": "@stencil/core",
-  "version": "2.8.1",
+  "version": "2.8.2-dryrun",
   "license": "MIT",

```
`package-lock.json`:
```diff
 {
   "name": "@stencil/core",
-  "version": "2.8.1",
+  "version": "2.8.2-dryrun",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "@stencil/core",
-      "version": "2.8.1",
+      "version": "2.8.2-dryrun",
       "license": "MIT",

```